server/pages/index.jade

```diff
@@ -128,18 +128,11 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 			script(type='text/javascript')!= config
 		if i18nLocaleScript
 			script(src=i18nLocaleScript)
-		if 'development' === env || isDebug
-			script(src=urls[ 'manifest' ])
-			script(src=urls[ 'vendor' ])
-			script(src=urls[ jsFile ])
-			if chunk
-				script(src=urls[ chunk ])
-		else
-			script(src=urls[ 'manifest-min' ])
-			script(src=urls[ 'vendor-min' ])
-			script(src=urls[ jsFile + '-min' ])
-			if chunk
-				script(src=urls[ chunk + '-min' ])
+		script(src=urls[ 'manifest' ])
```

Comment:
We need to make sure urls of JS files served on production remain the same. Otherwise returning visitors will have to load them again.